mu4/domain/notation/internal/igetscore.h

```diff
@@ -31,7 +31,7 @@ class IGetScore
 public:
     virtual ~IGetScore() = default;
 
-    virtual Ms::Score* score() const = 0;
+    virtual Ms::MasterScore* masterScore() const = 0;
```

Comment:
Maybe we should leave them both?
masterScore() and currentScore()

Reply:
For two reasons:
* I needed methods from MasterScore
* So it is more correct, at the moment we work only with MasterScore and all methods are designed for the fact that we are working with MasterScore.
When we implement the parts (Excerpts), then it will be clear where to transfer MasterScore and where arbitrary Score